src/pages/user-profile/profile.module.scss

```diff
@@ -0,0 +1,63 @@
+.userInfo {
+  margin-top: 5%;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  @media(max-width: 800px){
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+  }
+}
+
+.wrapper {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  justify-items: end;
+  grid-template-columns: 20% 20%;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  font-weight: bold;
+  @media(max-width: 800px){
+    grid-template-columns: 1fr;
+    justify-items: center;
+  }
+}
+
+p:nth-child(2n) {
```

Comment:
I would say to add `p:nth-child(2n), a:nth-child(2n) { ... }` and delete the next block (or leave just the `text-decoration: none` on the next one)